assigner/baserepo.py

```diff
@@ -176,6 +176,10 @@ def clone_to(self, dir_name, branch=None):
             if branch:
                 self._repo = git.Repo.clone_from(self.ssh_url, dir_name,
                                                 branch=branch)
+                for b in branch:
+                    self._repo.create_head(b, 'origin/{}'.format(b))
```

Comment:
np: we should stick to double quotes, I think that's what the rest of the file is using.